code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -130,49 +175,106 @@
 	layer = initial(layer) + piping_layer / 1000
 	return ..()
 
+/**
+ * check if a node can actually exists by connecting to another machine
+ * called on atmosinit()
+ * Arguments:
+ * * obj/machinery/atmospherics/target - the machine we are connecting to
+ * * iteration - the current node we are checking (from 1 to 4)
+ */
 /obj/machinery/atmospherics/proc/can_be_node(obj/machinery/atmospherics/target, iteration)
 	return connection_check(target, piping_layer)
 
-//Find a connecting /obj/machinery/atmospherics in specified direction
+/**
+ * Find a connecting /obj/machinery/atmospherics in specified direction, called by relaymove()
+ * used by ventcrawling mobs to check if they can move inside a pipe in a specific direction
```

Comment:
If it's only used as a part of one proc chain this is good, but optimally if I add a new call findConnecting somewhere I shouldn't need to update this comment to make things correct again. Remember, comments don't compile.

Reply:
mm ok what shoud i do then? remove what calls it?

Reply:
Well if it is only used for one particular proc chain, you should keep it, but if it's something that's used or is likely to be used in more then one spot, don't.